base_multi_image/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Multiple Images Base
+====================
+
+This module extends the functionality of any model to support multiple images
+(a gallery) attached to it and allow you to manage them.
+
+Installation
+============
+
+This module adds abstract models to work on. Its sole purpose is to serve as
+base for other modules that implement galleries, so if you install this one
+manually you will notice no change. You should install any other module based
+on this one and this will get installed automatically.
+
+Usage
+=====
+
+To manage all stored images, you need to:
+
+* Go to *Settings > Configuration > Multi images*.
+
+... but you probably prefer to manage them from the forms supplied by
+submodules that inherit this behavior.
+
+Development
+===========
+
+To develop a module based on this one:
+
+* See module ``product_multi_image`` as an example.
+* You have to inherit model ``base_multi_image.owner`` to the model that needs
+  the gallery::
+
+    class MyOwner(models.Model):
+        _name = "mymodule.name"
+        _inherit = "base_multi_image.owner"
+
+        # If you need this, you will need ``post_init_hook_for_submodules``
+        old_image_field = fields.Binary(related="image_main", store=False)
+
```

Comment:
Can you also offer here the XML we would typically add on the Forms wherev this feature is added?